web-app/src/components/ServicesPage/index.js

```diff
@@ -2,60 +2,51 @@ import React, { useState, useEffect } from "react";
 
 import { Box, Container } from "@material-ui/core";
 import { ToggleButton, ToggleButtonGroup } from "@material-ui/lab";
+import { GoogleApiWrapper, Map, Marker } from "google-maps-react"
 
 import Service from "./Service";
+import { myServices } from "../../constants/services"
 
-const ServicesPage = () => {
-  //initialstate
-  const myServices = [
-    {
-      title: "Provider 1",
-      address: "123 Main St",
-      services: ["Food", "Jobs"],
-    },
-    {
-      title: "Provider 2",
-      address: "411 E Wisconsin Ave",
-      unit: "9th Fl",
-      zip: "53202",
-    },
-    {
-      title: "Provider 3",
-      address: "808 N 3rd St",
-      city: "Milwaukee",
-      state: "WI",
-      zip: "53203",
-    },
-  ];
```

Comment:
moved these to a constants folder so we have a more scalable way of adding more services